airbyte-integrations/connectors/destination-snowflake/build.gradle

```diff
@@ -30,6 +30,7 @@ dependencies {
     implementation 'org.apache.commons:commons-csv:1.4'
     implementation 'com.github.alexmojaki:s3-stream-upload:2.2.2'
     implementation "io.aesy:datasize:1.0.0"
+    implementation 'com.zaxxer:HikariCP:5.0.1'
```

Comment:
why do we need this new dependency?

Reply:
I had to switch a connection's pool lib to be able to update the connection's config on a flight. -> from time to time get a new access token and put it connection's pool configs.  The current lib for connections pool doesn't allow it, or at least I didn't find a way.